Xamarin.PropertyEditing.Mac/Controls/Custom/CommonBrushLayer.cs

```diff
@@ -68,6 +70,7 @@ public static CALayer CreateBrushLayer (CommonBrush brush)
 		public override void LayoutSublayers ()
 		{
 			base.LayoutSublayers ();
+			Frame = new CGRect (Frame.X, Bounds.Y + VerticalMargin + 0.5, Bounds.Width, Bounds.Height - VerticalMargin * 2);
```

Comment:
Why the extra 0.5 on only one side?

Reply:
Because we're not adjusting the horizontal position at all here, just vertical (the color preview is vertically centered, but horizontally left-aligned), and vertical happens to be "naturally" (by which I mean if you let it do what it wants to do) one more pixel on one size than on the other. The 0.5 pixel restores parity between the margins above and below.